all: re-enable windows build

The tests are failing so I temporarily disabled the windows build.

The tests are failing with extraneous '\r` characters in wire generated files.

```
--- FAIL: TestWire (286.20s)
    --- FAIL: TestWire/BindInjectorArg (20.20s)
        wire_test.go:163: wire output differs from golden file. If this change is expected, run with -record to update the wire_gen.go file.
            *** got:
            // Code generated by Wire. DO NOT EDIT.
            
            //go:generate wire
            //+build !wireinject
            
            package main
            
            // Injectors from wire.go:
            
            func inject(foo *Foo) *Bar {
            	bar := NewBar(foo)
            	return bar
            }
            
            
            *** want:
            // Code generated by Wire. DO NOT EDIT.
            
            //go:generate wire
            //+build !wireinject
            
            package main
            
            // Injectors from wire.go:
            
            func inject(foo *Foo) *Bar {
            	bar := NewBar(foo)
            	return bar
            }
            
            
            *** diff:
            {[]string}[0]:
            	-: "// Code generated by Wire. DO NOT EDIT."
            	+: "// Code generated by Wire. DO NOT EDIT.\r"
            {[]string}[1]:
            	-: ""
            	+: "\r"
            {[]string}[2]:
            	-: "//go:generate wire"
            	+: "//go:generate wire\r"
            {[]string}[3]:
            	-: "//+build !wireinject"
            	+: "//+build !wireinject\r"
            {[]string}[4]:
            	-: ""
            	+: "\r"
            {[]string}[5]:
            	-: "package main"
            	+: "package main\r"
            {[]string}[6]:
            	-: ""
            	+: "\r"
            {[]string}[7]:
            	-: "// Injectors from wire.go:"
            	+: "// Injectors from wire.go:\r"
            {[]string}[8]:
            	-: ""
            	+: "\r"
            {[]string}[9]:
            	-: "func inject(foo *Foo) *Bar {"
            	+: "func inject(foo *Foo) *Bar {\r"
            {[]string}[10]:
            	-: "\tbar := NewBar(foo)"
            	+: "\tbar := NewBar(foo)\r"
            {[]string}[11]:
            	-: "\treturn bar"
            	+: "\treturn bar\r"
            {[]string}[12]:
            	-: "}"
            	+: "}\r"
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -15,6 +15,7 @@
 os:
   - linux
   - osx
+  - windows
 
 language: go
 go_import_path: github.com/google/wire
@@ -33,9 +34,13 @@ before_install:
 install:
   # Re-checkout files preserving line feeds. This prevents Windows builds from
   # converting \n to \r\n.
-  - "git config --global core.autocrlf input"
-  - "git checkout -- ."
-  - "go install ./cmd/wire"
+  - if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then
+      cd ../..;
+      mv $TRAVIS_REPO_SLUG _old;
+      git config --global core.autocrlf false;
+      git clone _old $TRAVIS_REPO_SLUG;
+      cd $TRAVIS_REPO_SLUG;
+    fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       go install github.com/mattn/goveralls;
     fi

